pkg/version/update.go

```diff
@@ -61,6 +61,14 @@ func CheckForUpdate(
 	}
 	q.Set("clientID", clientID)
 	q.Set("InstallationID", InstallationID)
+
+	// The BACALHAU_UPDATE_CHECKER_TEST is an env variable a user can set so that we can track
+	// when the binary is being run by a non-user, to enable easier filtering of queries
+	// to their update server for internal/CI.
```

Comment:
Can I assume this has some relation to: https://github.com/bacalhau-project/getbacalhauorg/pull/29 ?

> when the binary is being run by a non-user

What does this mean? Like a linux user or a person who uses bacalhau?

Reply:
person who uses / deploys / etc bacalhau.

Basically just let users set (if they want) a key indicating this run is "internal".